src/utils/dates.js

```diff
@@ -10,3 +12,101 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
```

Comment:
`getFutureDate` is bit ambiguous naming and it's unclear what the function does and what the input types are. By renaming to `addDaysFromToday` (open to suggestions for this one), it's clear that this method is adding x number of days to today's date. By renaming the input to `daysOffset`, it makes it more clear that you're supposed to be passing in the number of days.

```suggestion
export function addDaysFromToday(daysOffset) {
	return new Date(Date.now() + daysOffset * ONE_DAY_IN_MILLISECONDS);
}
```